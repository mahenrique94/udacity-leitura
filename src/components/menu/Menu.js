import React from 'react'

import { Navbar } from 'reactstrap'

import MenuLinks from './MenuLinks'
import MenuTitle from './MenuTitle'

const Menu = () => (
    <Navbar color="dark" dark data-cy="menu" expand="md" light>
        <MenuTitle/>
        <MenuLinks/>
    </Navbar>
)

export default Menu
