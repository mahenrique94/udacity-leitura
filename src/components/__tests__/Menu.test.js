import React from 'react'
import renderer from 'react-test-renderer'

import Menu from '../menu/Menu'

describe('Tests for Menu component', () => {

    it('Should rendering without crash', () => {
        const tree = renderer.create(
            <Menu/>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })

})
