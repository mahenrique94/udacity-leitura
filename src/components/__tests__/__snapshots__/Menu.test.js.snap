// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Menu component Should rendering without crash 1`] = `
<nav
  className="mb-5 navbar navbar-expand-md navbar-light navbar-dark bg-dark"
  data-cy="menu"
>
  <a
    href="/"
  >
    leitura
  </a>
  <button
    className="navbar-toggler"
    onClick={[Function]}
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    style={Object {}}
  >
    <ul
      className="ml-auto navbar-nav"
      data-cy="nav"
    >
      <li
        className="dropdown nav-item"
        data-cy="nav-languages"
        onKeyDown={[Function]}
      >
        <a
          aria-expanded={false}
          aria-haspopup={true}
          className="dropdown-toggle nav-link"
          data-cy="nav-languages-link"
          href="#"
          onClick={[Function]}
        >
          Language
        </a>
        <div
          aria-hidden={true}
          className="dropdown-menu dropdown-menu-right"
          role="menu"
          tabIndex="-1"
        >
          <button
            className="dropdown-item"
            data-cy="nav-languages-english"
            onClick={[Function]}
            tabIndex="0"
            type="button"
          >
            English
          </button>
          <div
            className="dropdown-divider"
            onClick={[Function]}
            tabIndex="-1"
          />
          <button
            className="dropdown-item"
            data-cy="nav-languages-portuguese"
            onClick={[Function]}
            tabIndex="0"
            type="button"
          >
            Portuguese
          </button>
        </div>
      </li>
    </ul>
  </div>
</nav>
`;
